### ISSUE TITLE
custom fields ;  with CustomFieldTemplate : weird behavior

### ISSUE THREAD

**Issue by vincentsixty** [mozilla-services/react-jsonschema-form#738]
this is my schema:
     
const schema = {

                    type: "object",
                    properties: {
                        rows: {
                            type: "array",
                            title: "Rows",
                            items: {
                                type: "array",

                                items: {

                                    type: "object",
                                    properties : {
                                        col: {
                                            type: "object",
                                            properties: {
                                                fn:{type: "string"}
                                            }
                                        }

                                        }
                                    }

                                }

                            }
                        },
          
        };

this is my ui schema:

        const uiSchema ={

             rows: {
                 items: {
                     items: {

                            col: {

                                   "ui:field": "geo4"

                            }
                     },

I create a customfieldTemplate for this form as followings;
function CustomFieldTemplate(props) {
    const {id, classNames, label, help, required, description, errors, children} = props;

   // if(_.isObject(_.get(props, 'schema.properties'))&&(_.keys(props.schema.properties) == "rows")||(_.isObject(_.get(props, 'schema.items.properties'))&&(_.keys(props.schema.properties) == "col")))
   // { return(<div>{children}</div>) ; }
  //  if(_.isObject(_.get(props, 'schema.items.properties'))&&(_.keys(props.schema.properties) == "col"))
  //  { return(<div></div>) ; }
      //else
  /* if( props.children.type.name === "TimeStampEditor2") {
        return(
            <div className="panel-body">
                {children}
            </div>  )
    }

Basically what happens is that This runs *Multiple* times , meaning it runs everytime for a custom and a non custom , and both inbetween ....
if i have html elements in this function they get created at least 4 times

**Comment by revolunet** [comment]
please take a minute to format the issue correctly, this is barely readable

thanks !
